docs/templating/README.md

```diff
@@ -100,15 +100,15 @@ export async function getServerSideProps(
 ): Promise<void>;
 ```
 
-These functions can be used to make additional requests using the Apollo Client that will be executed when Next.js [Data Fetchers](https://nextjs.org/docs/basic-features/data-fetching) run.
+Use these functions to make additional requests using the Apollo Client, which executes with Next.js [Data Fetchers](https://nextjs.org/docs/basic-features/data-fetching) run.
 
-You may find this useful if you find that you need additional data from the WordPress backend that isn't otherwise fetched by default.
+You may find this useful when you need additional data from the WordPress backend that doesn't fetch by default.
```

Comment:
```suggestion
You may find this useful when you need additional data from the WordPress backend that is not fetched by default.
```

Reply:
I am going to change this back to my original commit since "not fetched" is in a passive voice.